features/closewatcher.yml

```
@@ -0,0 +1,11 @@
+name: CloseWatcher
+description: "The CloseWatcher API listens and responds to close requests for developer-implemented user interface components such as sidebars or custom keyboards."
```

The examples here blur the lines a bit between browser/system UI and things happening inside the confines of the viewport. What do you think of this instead?

```suggestion
description: "The `CloseWatcher` API listens and responds to close requests on in-page components, such as modals, popovers, and sidebars."
```